### DISCUSSION TITLE
Multidocs & Algolia/DocSearch

### DISCUSSION THREAD

**Discussion by Dannyholt** [facebook/docusaurus#9254] [Q&A] [answered]
Hi,

I'm running multidocs for each of our companies products. I've renamed the main /docs instance to a product name using:

```js
{
  docs: {
    path: "product-a",
    routeBasePath: "product-a",
```

The problem is when searching, the product name doesn't display in DocSearch, instead it uses "Documentation". Is there a way to rename this?

<img width="555" alt="CleanShot 2023-08-23 at 22 24 40@2x" src="https://github.com/facebook/docusaurus/assets/1708426/dc4ac8f4-0855-41be-bcfa-0ce1f901ac79">

**Comment by slorber** [comment, marked as answer]
You have to make sure your crawler config is up to date, as well as the index (you can update it manually through the UI or delete it once you have updated your crowler config.

See also https://github.com/facebook/docusaurus/issues/9200 for another user facing a probably similar issue.
